src/control/system/member.go

```diff
@@ -36,13 +36,18 @@ const (
 	MemberStateStopping MemberState = 0x0010
 	// MemberStateStopped indicates process has been stopped.
 	MemberStateStopped MemberState = 0x0020
-	// MemberStateEvicted indicates rank has been evicted from DAOS system.
+	// MemberStateEvicted indicates rank has been automatically evicted from DAOS system.
 	MemberStateEvicted MemberState = 0x0040
 	// MemberStateErrored indicates the process stopped with errors.
 	MemberStateErrored MemberState = 0x0080
 	// MemberStateUnresponsive indicates the process is not responding.
 	MemberStateUnresponsive MemberState = 0x0100
+	// MemberStateExcluded indicates that the rank has been administratively excluded.
+	MemberStateExcluded MemberState = 0x0200
 
+	// ExcludedMemberFilter defines the state(s) to be used when determining
+	// whether or not a member should be excluded from CaRT group map updates.
+	ExcludedMemberFilter = MemberStateAwaitFormat | MemberStateEvicted | MemberStateExcluded
```

Comment:
AwaitFormat should only ever be set locally, i.e. the MS wont reflect this date for remote server instances afaik, so not sure it's relevant in the filter

Reply:
My thinking here was that at some point it might somehow get set in the MS DB (e.g. an engine is individually reformatted) and it seems like an ambiguous state. The engine might get the same rank or it might get a new one, in which case we shouldn't include the old rank in the system map.